charts/consul/templates/sync-catalog-deployment.yaml

```diff
@@ -196,6 +203,18 @@ spec:
             {{- if .Values.syncCatalog.syncLoadBalancerEndpoints }}
             -sync-lb-services-endpoints=true \
             {{- end }}
+            {{- if (or (and (ne (.Values.syncCatalog.metrics.enabled | toString) "-") .Values.syncCatalog.metrics.enabled) (and (eq (.Values.syncCatalog.metrics.enabled | toString) "-") .Values.global.metrics.enabled)) }}
+            -enable-metrics=true \
+            {{- else }}
+            -enable-metrics=false \
```

Comment:
so if we make this a boolean instead of a string we can get rid of the entire branch and just leave it unset

Reply:
can you expand on this comment, not sure what you mean by `we can get rid of the entire branch`